scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1550,6 +1570,22 @@
     (is (= (:uuid response-2) (get (first (list-jobs-fn submit-ms-1 (inc submit-ms-2))) "uuid")))
     (is (= (:uuid response-1) (get (second (list-jobs-fn submit-ms-1 (inc submit-ms-2))) "uuid")))))
 
+(deftest test-list-jobs-include-custom-executor
+  (let [conn (restore-fresh-database! "datomic:mem://test-list-jobs-include-custom-executor")
+        handler (basic-handler conn)
+        before (t/now)
+        list-jobs-fn #(list-jobs-with-jobs handler "user" ["running" "waiting" "completed"]
+                                           (.getMillis before) (.getMillis (t/now)))
+        response-1 (submit-job handler "user")
+        response-2 (submit-job handler "user")
+        _ @(d/transact conn [[:db/add [:job/uuid (UUID/fromString (:uuid response-1))] :job/custom-executor true]])
+        jobs (list-jobs-fn)]
+    (is (= 201 (:status response-2)))
+    (is (= 201 (:status response-1)))
+    (is (= 2 (count jobs)))
+    (is (= (:uuid response-2) (-> jobs first (get "uuid"))))
+    (is (= (:uuid response-1) (-> jobs second (get "uuid"))))))
```

Comment:
Should we assert on the `:job/custom-executor` flags of the two jobs also?

Reply:
Done